sqlglot/tokens.py

```diff
@@ -522,6 +526,7 @@ class Tokenizer(metaclass=_Tokenizer):
         "TRUNCATE": TokenType.TRUNCATE,
         "UNBOUNDED": TokenType.UNBOUNDED,
         "UNION": TokenType.UNION,
+        "UNPIVOT": TokenType.UNPIVOT,
```

Comment:
you need to add all these new tokens to id_vara

Reply:
and also because they are query modifiers, need to make them special, but they shouldn’t be allowed in dialects that don’t support or else you won’t be able to support 

select from x pivot where pivot is the alia’s